jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/impl/Bpmn2JsonMarshaller.java

```diff
@@ -899,9 +910,7 @@ private void setThrowEventProperties(ThrowEvent event,
                 }
             } else if (ed instanceof MessageEventDefinition) {
                 if (((MessageEventDefinition) ed).getMessageRef() != null) {
```

Comment:
We don't need this `if`, as it is done inside `setMessageRefProperties` implementation.